ludwig/schema/llms/peft.py

```diff
@@ -69,6 +69,18 @@ class LoraConfig(BaseAdapterConfig):
         description="Bias type for Lora.",
     )
 
+    pretrained_weights: Optional[str] = schema_utils.String(
+        default="none",
```

Comment:
Should this be `None`? If so, you can pass `allow_none=True` to `schema_utils.String` so that it doesn't raise an exception.

Reply:
I was going by the convention that Travis used on line 68, but I can change this.

Reply:
Ah, no worries! If it makes sense there then I wouldn't change it.